### PR TITLE
Add README and improve en-US translations

### PR DESCRIPTION
**Pull request by Tursko**
Added README with basic documentation (everything I know at least) and improved the English translations (as a native english speaker).

### CODE REVIEW

File: language_en-US.js [hunk 1]
```diff
@@ -56,8 +56,8 @@ module.exports = {
       REMOTE_CONTROL_DISALLOW: "The context cannot be remote-controlled.",
       DEVICE_NOT_CONTROLLABLE: "It is not possible to remote control the device.",
       VOLUME_CONTROL_DISALLOW: "It is not possible to remote control the device’s volume.",
```
Comment (NicoWeio) [inline language_en-US.js]: Speaking of contractions, how about this one?
```suggestion
      DEVICE_NOT_CONTROLLABLE: "It's not possible to remote control the device.",
      VOLUME_CONTROL_DISALLOW: "It's not possible to remote control the device’s volume.",
```

Comment (NicoWeio) [reply language_en-US.js]: Overall, it's looking good! Will merge once this nitpick is sorted out.

File: language_en-US.js [hunk 2]
```diff
@@ -57,7 +57,7 @@ module.exports = {
...
       NO_ACTIVE_DEVICE: "I don't see an active device to perform this action on.",
-      PREMIUM_REQUIRED: "Unfortunately, this request is prohibited for non-Premium users.",
+      PREMIUM_REQUIRED: "Unfortunately, you need a Spotify Premium account to do this.",
```
Comment (Tursko) [inline language_en-US.js:60]: This sounds much better.

File: language_en-US.js (hunk 1 above)
Comment (Tursko) [reply language_en-US.js]: Both "It's not" and "It isn't" work in this scenario, but I think emphasizing the word "not" is good. So sure, "It's not" sounds good.

Comment (Tursko) [reply language_en-US.js]: https://github.com/NicoWeio/Alexa-Spotify-Remote/pull/4/commits/200da3d302905a7aefdb4147bd94799249dfd344

**Pull request merged**